src/aap_eda/core/models/project.py

```diff
@@ -68,6 +70,9 @@ class ScmType(models.TextChoices):
         default=None,
         on_delete=models.SET_NULL,
     )
+    organization = models.ForeignKey(
```

Comment:
@Dostonbek1 @benthomasson Can we use simple inheritance from a base model that has the ForeignKey for organization, if we add a new model down the road and it needs to have Organization in it we can inherit from the base model. Then we don't need this in every class. We had done this in catalog with Tenants which is needed in majority of our objects. https://github.com/ansible/pinakes/blob/122fc3184b0c6b6745de220bbd42e632531ec93b/pinakes/main/models.py#L49

Reply:
Sure.  This is a good use of inheritance in Django, but we should try to keep the inheritance hierarchy to two levels:   
base models and models.